webapp/packages/plugin-authentication/src/AuthenticationService.ts

```diff
@@ -106,7 +107,21 @@ export class AuthenticationService extends Bootstrap {
     }
 
     try {
-      await this.userInfoResource.logout(providerId, configurationId);
+      // TODO handle all redirect links once we know what to do with multiple popups issue
+      const {
+        authLogoutExtended: { redirectLinks },
+      } = await this.userInfoResource.logout(providerId, configurationId);
+
+      if (redirectLinks.length) {
+        const oktaLink = redirectLinks[0];
+        const id = `okta-logout-id-${uuid()}`;
+
+        this.windowsService.open(id, {
+          url: oktaLink,
+          width: 400,
+          height: 400,
+        });
+      }
```

Comment:
this logic must replace 
```ts
if (userAuthConfiguration?.configuration.signOutLink) {
      this.logoutConfiguration(userAuthConfiguration);
    }
```

we need to rewrite or remove it or change and maybe to change `logoutConfiguration`